apps/docs/components/ui/subHeader/subHeader.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import SectionPopover from "@/components/ui/sectionPopover/sectionPopover";
+
+import "./subHeader.css";
+
+interface Link {
+    title: string;
+    url: string;
+    id: string;
+}
+
+interface SubHeaderProps {
+    isOpen: boolean;
```

Comment:
this props does not seem to be used

Reply:
It's used in the token layout file.

Reply:
What I mean is that it's not passed in the component props nor used in the SubHeader

Reply:
Yep I answered too fast corrected.